packages/@uppy/audio/src/formatSeconds.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Takes an Integer value of seconds (e.g. 83) and converts it into a human-readable formatted string (e.g. '1:23').
+ *
+ * @param {Integer} seconds
+ * @returns {string} the formatted seconds (e.g. '1:23' for 1 minute and 23 seconds)
+ *
+ */
+module.exports = function formatSeconds (seconds) {
```

Comment:
Might be worth to add a TODO comment to switch to https://tc39.es/proposal-temporal/docs/duration.html#toLocaleString when it's implemented on all supported browsers.

Reply:
Alternatively, use [`Intl.DateTimeFormat`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat)